vite.config.mts.timestamp-1737379816854-14baa337b2fe5.mjs

```
@@ -0,0 +1,307 @@
+    url: `https://${org}.github.io/${repo}/assets/remoteEntry.js`,
+    org,
+    repo
+  };
+}
+function getRemotes(enabledApps) {
+  if (!enabledApps) return {};
+  return enabledApps.split(",").reduce((acc, app) => {
+    const { repo, url } = parseRemoteConfig(app);
+    console.log(`Configuring Remote Module for ${repo}:`, url);
+    return {
+      ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)